### DISCUSSION TITLE
Switches DNDMode

### DISCUSSION THREAD

**Discussion by benderunit22** [krahabb/meross_lan#125] [Q&A] [answered]
Hi, sorry if this is a topic that has been already covered.  First off, this integration is wonderful.  Works with all 12 of my Meross devices.  The question I have is in regards to the plugs.  I've noticed that when I add them, each has to entities.  One is the normal one to control and there is a second labeled DNDMode.  What is DNDMode and why is a entity created for it?  When I toggle them, doesn't seem to do anything.

**Comment by krahabb** [comment, marked as answer]
Hello @benderunit22 ,
the DNDMode is a feature to put the device in 'do not disturb' mode. This is usually represented by the light state of the device: when DND mode is ON the light goes off so the device doesn't 'disturb' the sorrounding ambient with it's presence.
In previous releases of meross_lan this was really working like this so it was implement as a switch entity to activate DND mode (i.e. turn off the presence light)
With latest release (2.4.0 and so on) the logic has been aligned to the light state. Moreover the entity in HA is now a light and no more a switch so to better represent the common device behaviour.
Also, this entity is now a 'configuration entity' so it doesn't appear in default lovelace cards and is a bit 'hidden'. You can check this [official post](https://www.home-assistant.io/blog/2021/11/03/release-202111/#entity-categorization) about this new feature appearing in HA since 2021.11

**Reply by benderunit22** [reply]
Thank you so much for answering my question so thoroughly.  Great work, very appreciated.
